kitsune/search/api.py

```diff
@@ -11,49 +12,75 @@
 from kitsune.wiki.models import DocumentMappingType
 
 
+def positive_integer(value):
```

Comment:
The serializer has it as an IntegerField, so that should do the integer validation. Pretty sure one of the existing tests covers that scenario.